runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -129,7 +129,7 @@ public HiscorePanel()
 
 		input = new IconTextField();
 		input.setIcon(search);
-		input.setFont(labelFont.deriveFont(Font.BOLD));
+		input.setFont(labelFont);
```

Comment:
You can just remove the labelFont completely here, as the default one is exactly same, previous logic was just making it bold that is why it is here.